### PR TITLE
Makefile cleanups.

### PR DESCRIPTION
**Pull request by julialongtin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,6 +51,7 @@ i:
 	$(MAKE) -C services/brig i
 	$(MAKE) -C services/gundeck i
 	$(MAKE) -C services/spar i
+	$(MAKE) -C libs/cassandra-util i
```
Comment (fisx) [inline Makefile]: hum, this is not a service.  why do you want to build it here?

**Comment by julialongtin** [comment]
sorry, this snuck in.

File: services/restund/Makefile [hunk 2]
```diff
@@ -18,6 +18,8 @@ WIRE_VERSION    := 6
 EXTRA_MODULES   := zrest drain
 
 DOCKER          ?= true
+# FIXME: use quay.io/wire
```
Comment (ChrisPenner) [inline services/restund/Makefile]: ☝️

Comment (julialongtin) [reply services/restund/Makefile]: The other Makefiles use quay.io/wire . This Makefile is an outlier. I wanted to make it clear something was strange, while making a minor improvement by allowing us to change the username, and test.

**Pull request merged**